src/EnergyPlus/PlantLoopEquip.cc

```diff
@@ -308,19 +307,19 @@ namespace PlantLoopEquip {
 		//Pipe has no special types at the moment, so find it this way
 		if ( GeneralEquipType == GenEquipTypes_Pipe ) {
 			if ( EquipTypeNum == TypeOf_Pipe ) {
```

Comment:
Yes, actually my hope is to eliminate this file entirely.  Again, this was somewhat delayed to show the devs the before and after of each individual component.  As things inherit PlantComponent, this will all diminish away...